sambamba/pileup.d

```diff
@@ -257,7 +262,7 @@ string[] unbundle(string[] args, string exclude="") {
 
 // input: unbundled samtools arguments
 // output: detected output format
-FileFormat fixSamtoolsArgs(ref string[] args, bool use_caller) {
+FileFormat fixSamtoolsArgs(bool use_bcftools, ref string[] args) {
```

Comment:
By naming the parameter `use_caller` I tried to imply that other callers might be supported in the future. Could you please rename it back?